packages/dev/cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Do we need this file at all?

Reply:
It turns out we do need this file. Without it Cypress can't resolve `/commands`. See [here](https://github.com/f5/unovis/actions/runs/10203526794/job/28230045052)